app/src/main/java/ch/epfl/sdp/social/ChatRepository.java

```diff
@@ -138,14 +138,14 @@ public static void getMessagesReceived(final String id_owner, final String id_re
                 List<Message> msgList = new LinkedList<>();
                 //msgList.addAll(singleton.chatDB.daoAccess().getMessagesFromOwnerToReceiver(id_owner, id_rec));
                 //msgList.add(new Message(new Date(), "8*^&=*%^&*()90")); // canary value (temporary solution for now)
-                msgList.addAll(singleton.chatDB.daoAccess().getMessagesToOwnerFromSender(id_owner, id_rec));
+                msgList.addAll(singleton.chatDB.daoAccess().getMessagesToOwnerFromSender(id_rec, id_owner));
```

Comment:
Since the method says "To Owner From Sender" i.e. it specifies the owner first, then I think having the id of the owner come first in the argument list is more intuitive. Either considering refactoring the the method name or change the query instruction of the ChatAccessor daoAcess object

Reply:
Yeah, I will refactor this